maven-plugin/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java

```
@@ -270,6 +277,137 @@ public String promptForPlatformVersion(List<String> versions) {
         return version;
 	}
 
+    @Override
+    public String promptForJdkPath(Server server) {
+
+        List<String> paths = new ArrayList<>();
+        paths.add("JAVA_HOME (currently: " + System.getProperty("java.home") + ")");
+        try {
+            paths.addAll(getJdkPathsFromProperties(getSdkProperties()));
+        } catch (IOException e) {
+            e.printStackTrace();
```

Never eat exceptions. Rethrow this one as IllegalArgumentException.